build_daemon/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+### 0.1.0
```

Comment:
really useless nit but I think two `##`s is better, that allows us to use the `###` for subheadings instead of `####` which is barely recognizable as a header. Also at least in github `##` has an underline which helps separate each version a bit better

## Two

### Three

####  Four